DashAI/front/src/components/experiments/RunnerDialog.jsx

```diff
@@ -92,8 +83,19 @@ function RunnerDialog({ experiment, expRunning, setExpRunning }) {
       } else if (error.request) {
         console.error("Request error", error.request);
       } else {
-        console.error("Unkown Error", error.message);
+        console.error("Unknown Error", error.message);
       }
+    } finally {
+      enqueueSnackbar(`${experiment.name} has finished running`, {
+        variant: "info",
+        anchorOrigin: {
+          vertical: "top",
+          horizontal: "right",
+        },
+      });
+      setExpRunning({ ...expRunning, [experiment.id]: false });
```

Comment:
What does `[experiment.id]: false ` do?

Reply:
`expRunning` is a state that contains information about the execution status of all experiments in the table.

What that line does is indicate that the experiment has finished running, whether it ended successfully or ocurred an error.